sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 11.3.0-beta.3 (Unreleased)
+## 11.3.0 (2021-06-08)
 
+### Changed
+- This is a GA release to match the features that are currently available out of preview by the service.
+
+### Breaking Changes
+- Removed [`LexicalNormalizer`](https://docs.microsoft.com/azure/search/search-normalizers#predefined-normalizers) and the `Normalizer` [`SearchField`](https://docs.microsoft.com/dotnet/api/azure.search.documents.indexes.models.searchfield) from index definitions.
+- Removed support for [Semantic Search](https://docs.microsoft.com/azure/search/semantic-search-overview).
```

Comment:
It feels kind of weird to list these as breaking changes?  They're not gone, they're just not GA'ed yet based on the service.

Reply:
Sort of agree. They have been removed from the latest beta of this release train (11.3.0-beta.2). So, users on the bleeding edge of the SDK will see it as features being removed. However, from the service's POV, this release maps to the current GA where these features were never introduced.
We are in a weird spot in that respect for this release. Do you think we should announce it under 'Changes' as 'Maps features to the latest version of the Search service'?

Reply:
And I see now I already have written the exact thing as I proposed above. :)
I will go ahead and remove the breaking changes section then? It will be implied that features not in service GA are not included.

Reply:
I generally think these need to be optimized for customers rather than us or the service.  Most people only look at GA versions so seeing a bunch of breaking changes here is probably going to be more confusing than helpful.  What if we went with something more like

> ### Added
> - Adds stable features and bug fixes from the 11.3.0-beta.1 and 11.3.0-beta.2 releases.  Preview service features not generally available yet, like Semantic Search and Normalizers, are not included in this GA release.

or any other verbiage to help communicate that we're forking off of 11.2.0 and adding stuff, not shipping the 11.3.0-beta.* releases as GA?